### PR TITLE
Handle non latin numbers in numbers check

### PR DESCRIPTION
**Pull request by unho**
References translate/pootle#4203

### CODE REVIEW

**Comment by phlax** [comment]
lgtm apart from failing tests

File: translate/filters/test_checks.py [hunk 1]
```diff
@@ -521,6 +521,54 @@ def test_numbers():
...
+
+def test_arabic_numbers():
+    """test non latin numbers for Arabic"""
+    ar_checker = checks.StandardChecker(checks.CheckerConfig(targetlanguage='ar'))
+    assert passes(ar_checker.endwhitespace, u"أقصى طول للوسم ٢٥ حرفًا", u"Tags are limited to 25 characters")
+    assert passes(ar_checker.endwhitespace, u"حركة ب&طيئة (٠٫٥×)", u"&Slow Motion (0.5×)")
+    assert passes(ar_checker.endwhitespace, u"متصفح &٣٦٠ الآمن", u"&360 Secure Browser")
+    assert passes(ar_checker.endwhitespace, u"طلب واحد، #2 ك.بايت، #3ث;طلبان، #2 ك.بايت، #3ث;#1 طلبات، #2 ك.بايت، #3ث;#1 طلبا، #2 ك.بايت، #3ث;#1 طلب، #2 ك.بايت، #3ث;لا طلبات", u"One request, #2 KB, #3 s;#1 requests, #2 KB, #3 s")
+    assert passes(ar_checker.endwhitespace, u"من <strong>كتاب موزيلا،</strong> ١٥‏:١", u"from <strong>The Book of Mozilla,</strong> 15:1")
+    assert passes(ar_checker.endwhitespace, u"١٧٥٪", u"175%")
```
Comment (dwaynebailey) [inline translate/filters/test_checks.py]: Surely these and should be ``.numbers`` checks not ``.endwhitespace``?

File: translate/lang/fa.py [hunk 2]
```diff
@@ -64,6 +64,29 @@ class fa(common.Common):
...
+        (u"1", u"١"),  # U+0661 Arabic-Indic digit one.
+        #(u"2", u"٢"),  # U+0662 Arabic-Indic digit two.
+        #(u"3", u"٣"),  # U+0663 Arabic-Indic digit three.
+        #(u"4", u"٤"),  # U+0664 Arabic-Indic digit four.
+        #(u"5", u"٥"),  # U+0665 Arabic-Indic digit five.
+        #(u"6", u"٦"),  # U+0666 Arabic-Indic digit six.
+        #(u"7", u"٧"),  # U+0667 Arabic-Indic digit seven.
+        #(u"8", u"٨"),  # U+0668 Arabic-Indic digit eight.
+        #(u"9", u"٩"),  # U+0669 Arabic-Indic digit nine.
+        (u"0", u"۰"),  # U+06F0 Extended Arabic-Indic digit zero.
```
Comment (dwaynebailey) [inline translate/lang/fa.py]: I'm not 100% sure why these ones are commented out?  Could you add a comment to explain that please.

**Comment by dwaynebailey** [comment]
@unho added some comments, otherwise lgtm

File: translate/filters/test_checks.py (hunk 1 above)
Comment (unho) [inline translate/filters/test_checks.py]: Right.

File: translate/lang/fa.py (hunk 2 above)
Comment (unho) [inline translate/lang/fa.py]: To quickly enable them in the future in case those are necessary. The uncommented one is because I found a real use of it.

Comment (unho) [inline translate/lang/fa.py]: Added a comment.

**Comment by unho** [comment]
I thought this was ready to be merged but it seems that the tests are not passing for Python 2.6.

**Comment by dwaynebailey** [comment]
@unho doesn't seem to be a test but is rather a syntax error on Python 2.6

**Comment by unho** [comment]
Yes, I can't trace it to any TTK code.

**Pull request merged**